nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/factory/BaseNDArrayFactory.java

```diff
@@ -197,7 +197,32 @@ public INDArray linspace(int lower, int upper, int num, DataType dtype) {
         for (int i = 0; i < num; i++) {
             double t = (double) i / (num - 1);
             data[i] = lower * (1 - t) + t * upper;
+        }
```

Comment:
Can we replace these BaseNDArrayFactory methods with a native op call?
Doesn't really make sense to populate in java IMO, especially for CUDA.

Reply:
It shouldn't be used anywhere now i think? I'd expect ops to be used for linspace everywhere from now on

Reply:
Hm... maybe we remove linspace methods from BaseNDArrayFactory interface entirely then?

Reply:
That's what I'm doing, no need for Java linspace implementation anymore.

Reply:
Fixes for all issues were pushed.